Challenge1/README.md

```diff
@@ -12,14 +12,22 @@ This bot detects Forta bots deployed by Nethermind.
 ## Alerts
 
 - NEW-BOT-DEPLOYED
+
   - Fired when the `createAgent` function in the Forta Registry contract is called by the Nethermind EOA, i.e. a new Forta Bot is deployed by Nethermind
   - Severity is always set to "low"
   - Type is always set to "info"
   - Metadata: agentId (string)
 
-## Test Data
+- EXISTING-BOT-UPDATED
+  - Fired when the `updateAgent` function in the Forta Registry contract is called by the Nethermind EOA, i.e. an existing Forta Bot is updated by Nethermind
+  - Severity is always set to "low"
+  - Type is always set to "info"
+  - Metadata: agentId (string)
 
-<!-- The bot behaviour can be verified with the following transactions:
+## Test Data
 
-The test transaction examples should follow the format: [//txn hash//](//link to txn hash in block explorer//)
-- 0x3a0f757030beec55c22cbc545dd8a844cbbb2e6019461769e1bc3f3a95d10826 (15,000 USDT) -->
+1. returns empty findings if transaction is not from Nethermind address
+2. returns empty findings if transaction is not to Forta address
+3. returns empty findings if neither createAgent nor updateAgent are detected
+4. detects bot deployment
+5. detects bot update
```

Comment:
If you take a look at our bots in the _Forta-Agents_ repo, we usually provide a transaction hash that the bot could be executed against to test its functionality that way. [Here](https://github.com/NethermindEth/Forta-Agents/tree/main/DYDX-Bots/Large-USDC-Staking-Withdrawals#test-data) is an example.